whitelist.txt

```diff
@@ -0,0 +1,162 @@
+0rc1
```

Comment:
Is it possible to move this file to the devtools folder and have a more specific name? Can we also use the phrase "inclusionlist" over "whitelist"?

Reply:
Renamed it to devtools\allowlist.txt.  It requires using following syntax in ci build.

```shell
flake8 --whitelist devtools\allowlist.txt parameters
```

It might be possible to specify this in setup.cfg.

I've also opened an issue with flake8-spellcheck to update their terminology.  

https://github.com/MichaelAquilina/flake8-spellcheck/issues/44